### PR TITLE
Barratry Update (w/ custom cargo shuttle) Take 2

### PR DESCRIPTION
**Pull request by spanky-spanky**
## About the PR
Further changes to Barratry including minor fixes and improvements such as moving the detective's office into security and adding a sauna. Also adds a custom cargo shuttle which is the base shuttle but Barratryized.

## Requirements

- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.

**Changelog**
skub

### CODE REVIEW

File: Resources/Prototypes/_Harmony/Maps/barratry.yml [hunk 1]
```diff
@@ -9,12 +9,14 @@
       stationProto: StandardNanotrasenStation
       components:
         - type: StationNameSetup
-          mapNameTemplate: '{0} Barratry {1}'
+          mapNameTemplate: '{0} Pioneer {1}'
```
Comment (FluffMe) [inline Resources/Prototypes/_Harmony/Maps/barratry.yml:12]: Why rename it?

Comment (spanky-spanky) [reply Resources/Prototypes/_Harmony/Maps/barratry.yml:12]: I have plans to develop a "clean" version of the station in future and Pioneer will be the name of the station for both maps. Just a precursor to that eventual development. I can undo if needed.

Comment (FluffMe) [reply Resources/Prototypes/_Harmony/Maps/barratry.yml:12]: No need. Just checking in case of bad copy paste

**Review by FluffMe — APPROVED** [review]
LGTM

**Pull request merged**